cypress-tests/integration-tests/features/autocomplete.feature

```diff
@@ -4,24 +4,49 @@ Feature: Autocomplete
     Given I navigate to the handler page
     And I click the Show Filter button
 
-  Scenario: Autocomplete filter is displayed
-    Then the Autocomplete box is shown
+  Scenario: Static autocomplete filter is displayed
+    Then the static Autocomplete box is shown
 
-  Scenario: Autocomplete options are displayed
-    When I enter text longer than the minimum data length
-    Then a list of matching options is displayed beneath the autocomplete box
+  Scenario: Static autocomplete options are displayed
+    When I enter text longer than the minimum data length into the static Autocomplete box
+    Then a list of matching options is displayed
 
-  Scenario: Autocomplete options are not displayed if the text is too short
-    When I enter text shorter than the minimum data length
+  Scenario: Static autocomplete options are not displayed if the text is too short
+    When I enter text shorter than the minimum data length into the static Autocomplete box
     Then a list of options is not displayed
 
-  Scenario: Autocomplete options are selectable
-    Given I enter text longer than the minimum data length
+  Scenario: Static autocomplete options are selectable
+    Given I enter text longer than the minimum data length into the static Autocomplete box
     When I select an autocomplete option
     Then that value is displayed in the autocomplete box
 
-  Scenario: Autocomplete options are submitted
-    Given I enter text longer than the minimum data length
+  Scenario: Static autocomplete options are submitted
+    Given I enter text longer than the minimum data length into the static Autocomplete box
+    And I select an autocomplete option
+    When I apply the filters
+    Then the selected option is displayed in the URL
+    And the select option is displayed in the Selected Filters section
+
+  Scenario: Dynamic autocomplete filter is displayed
+    Then the dynamic Autocomplete box is shown
+
+  Scenario: Dynamic autocomplete options are displayed
+    When I enter text longer than the minimum data length into the dynamic Autocomplete box
```

Comment:
Similar comment as above for the static autocomplete https://github.com/ministryofjustice/hmpps-digital-prison-reporting-frontend/pull/22/files#r1457619429